src/app/shared/case-editor/case-edit-page.component.ts

```diff
@@ -128,6 +123,9 @@ export class CaseEditPageComponent implements OnInit, AfterViewChecked {
 
   previous(): Promise<boolean> {
     this.error = null;
+    if (this.eventTrigger.can_save_draft) {
```

Comment:
this check, which is duplicated, could be moved inside 
this.saveDraft()

Reply:
addressed in the next [PR](https://github.com/hmcts/ccd-case-management-web/pull/183)